app/script/view_model/content/InputBarViewModel.js

```diff
@@ -524,6 +527,7 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
   }
 
   sendMessageEdit(messageText, messageEntity) {
+    const mentionEntities = [...this.currentMentions];
```

Comment:
Fancy but I think a simple `this.currentMentions.slice()` will do in this case. No need for spread operator.